.github/workflows/perf.yml

```diff
@@ -176,7 +171,7 @@ jobs:
           max-items-in-chart: ${{ env.MAX_BENCHMARKS_TO_KEEP }}
           gh-pages-branch: ${{ env.GH_PAGES_BRANCH }}
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          commit-sha: ${{ github.event.inputs.sha }}
+          commit-sha: ${{ needs.get-testing-version.outputs.commit_id }}
```

Comment:
How is this not the same value?

Reply:
With a `repository_dispatch` event, the SHA is in `github.event.client_payload.sha` instead of `github.event.inputs.sha`.